lib/vsc/ldap/utils.py

```diff
@@ -491,6 +491,9 @@ def __modify(self, current, dn, attributes):
         current_ = {}
         for key in attributes.keys():
             current_[key] = current.get(key, [])
+            if current_[key] is '':
+                self.log.warning("Replacing empty string for key %s with EMPTYGECOSDURINGMODIFY before making modlist for dn %s" % (key, dn))
+                current_[key] = "EMPTYGECOSDURINGMODIFY"  # hack to allow replacing empty strings
```

Comment:
Also, your earlier comment stated explicitly to "use a magic string that is relevant to the error like EMPTYGECOSDURINGMODIFY". So I am a bit confused as to why this is now not allowed.